src/Selenium2Library/keywords/_element.py

```diff
@@ -290,6 +290,33 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_height(self, locator):
+        """Returns hight of element identified by `locator`.
+            
+        The position is returned in pixels,as an integer.
```

Comment:
for Position there is Get Vertical Position and Get Horisontal Position.
These two return the coordinates for top-left corner.

with my requested methods users will be able to get size for an element.
This way, users will be able to to some elements style testing.

On Tue, Jun 23, 2015 at 8:10 PM, Zephraph <notifications@github.com> wrote:

> In src/Selenium2Library/keywords/_element.py
> <https://github.com/rtomac/robotframework-selenium2library/pull/430#discussion_r33065634>
> :
>
> > @@ -290,6 +290,33 @@ def get_horizontal_position(self, locator):
> >              raise AssertionError("Could not determine position for '%s'" % (locator))
> >          return element.location['x']
> >
> > +    def get_element_height(self, locator):
> > +        """Returns hight of element identified by `locator`.
> > +
> > +        The position is returned in pixels,as an integer.
>
> Is this supposed to be *position* instead of height?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomac/robotframework-selenium2library/pull/430/files#r33065634>
> .
>